test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -2809,27 +2923,28 @@ public void testAuthzAuthorityAcl_noResults() throws Exception {
   @Test
   public void testAuthzAuthorityAcl_permit() throws Exception {
     executeUpdate("create table acl(id integer, gsa_permit_users varchar(20))");
+    executeUpdate("create table data(id integer, author varchar(20))");
     executeUpdate("insert into acl(id, gsa_permit_users) values "
         + "(2, 'alice')");
 
     Map<String, String> moreEntries = new HashMap<String, String>();
     moreEntries.put("db.aclSql", "select * from acl where id = ?");
-    // Required for validation, but not specific to this test.
```

Comment:
Why did you remove this, or treat the data table, modeOfOperation, or everyDocIdSql as specific to this test? I can see moving db.uniqueKey and db.everyDocIdSql up (though I probably wouldn't have move the latter, since it's never executed).

Reply:
It looked odd to have "create table data.." after "moreEntries.put("db.aclSql"..", hence moved it to top. However, I see that the other tests have this pattern. Restored. it. Thx.